src/main/java/xyz/whoneedspacee/ssmos/managers/disguises/Disguise.java

```diff
@@ -338,7 +388,15 @@ public float getPitch() {
     }
 
     public void playDamageSound() {
-        owner.getWorld().playSound(owner.getLocation(), getDamageSound(), getVolume(), getPitch());
+        // gholemFix: similar to the damageUtil sound method fix, if its guardian, use NMS make sound with the string, else use bukkit api
```

Comment:
This should have been handled in the guardian disguise class, as of right now I don't think this even does anything?